test/cbmc/proofs/Socket/vSocketBind/vSocketBind_harness.c

```diff
@@ -55,8 +57,18 @@ void harness()
     __CPROVER_assume( pxSocket != NULL );
     __CPROVER_assume( pxSocket != FREERTOS_INVALID_SOCKET );
 
+    pxSocket->pxEndPoint = safeMalloc( sizeof( *( pxSocket->pxEndPoint ) ) );
+    __CPROVER_assume( pxSocket->pxEndPoint != NULL );
```

Comment:
Want to get rid of safeMalloc like you have other places?

Reply:
Will do.

Reply:
Well, now you have changed the semantics because (unless the cbmc flag is used) malloc will never return NULL.  I was coaching in the direction of using the conditional expression (that implements safemalloc).

Reply:
Oh. Something else was on my mind. Thanks, I updated